rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/YamlSerialization.cpp

```diff
@@ -85,7 +85,7 @@ rmf_traffic_msgs::msg::ConvexShape convex_shape(YAML::Node node)
 
   rmf_traffic_msgs::msg::ConvexShape shape;
   shape.type = shape_type(node[TypeKey]);
-  shape.index = node[IndexKey].as<uint8_t>();
+  shape.index = node[IndexKey].as<uint16_t>();
```

Comment:
The [message definition](https://github.com/open-rmf/rmf_internal_msgs/blob/c0b929db6ca666f7951dc633c573892a7c812c28/rmf_traffic_msgs/msg/ConvexShape.msg#L13) has this field as a `uint8`. Is there a reason to change the type to `uint16_t` here?